### ISSUE TITLE
Redundant onchain payreqs are necessary but we don't need to spam the logs about it

### ISSUE THREAD

**Issue by wtogami** [ElementsProject/peerswap#71]
Version: after #66 

LND peerswap
```
2022/06/03 03:40:55 [INFO]
Custom Message Handler err: Message already exists
2022/06/03 03:41:05 [DEBUG] [Messenger] From: REDACTED got msgtype: a45d payload: {"swap_id":"6a0fbd775103b48e08ac7bf3e873637cc539442ee1a64d05710fcc62e4a37560","payreq":"lnbc1REDACTEDBLINDEDLIWUIDADDRESS","tx_id":"REDACTEDTXID","script_out":1,"blinding_key":"REDACTEDBLINDINGKEY"}
```

CLN peerswap sees the same thing.
```
2022-06-03T00:10:55.417Z DEBUG   plugin-peerswap-plugin:  msghandler err: Message already exists
```

These redundant events are necessary but we don't need to spam logs about it. Only if something changes or times out?

**Comment by nepet** [comment]
Yes this is intended as we don`t know if the peer received the message or not. This is brute force. We could be more sophisticated by introducing `ack` messages to the protocol. https://github.com/ElementsProject/peerswap-spec

**Comment by nepet** [comment]
For documentation: the issue is about the log level and we should be less noisy about recurring actions

**Comment by wtogami** [comment]
Fixed by #203
